runtime/rasdump/javadump.cpp

```diff
@@ -1421,6 +1393,66 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	);
 }
 
+void
+JavaCoreDumpWriter::writeEnvUserArgsHelper(J9VMInitArgs *vmArgs)
+{
+	if (NULL == vmArgs) {
+		return;
```

Comment:
can you verify that `vm->checkpointState.restoreArgsList` exists if no options are specified on restore

Reply:
The second example given in the PR is what happens when no restore options are given, so `restoreArgsList` exists in that case but it's just empty.